packages/graphql-auth-transformer/package.json

```diff
@@ -7,7 +7,8 @@
     "test": "jest",
     "test-ci": "jest --ci -i",
     "build": "tsc",
-    "clean": "rm -rf ./lib"
+    "clean": "rm -rf ./lib",
```

Comment:
nit: rm is *nix only, we should use rimraf everywhere, since it will break on windows. Yes I know we're currently breaking dev on windows, but it would be nice to no to add new 'problems'.